dayliopy/clean_csv.py

```diff
@@ -153,7 +159,8 @@ def clean_daylio_data(available_activities: set, in_data: list[dict[str, str]]):
     Convert Daylio CSV file to a more usable CSV report.
     """
     default_activities = {key: 0 for key in available_activities}
-    out_data = [clean_row(row, default_activities.copy()) for row in in_data]
+    moods: dict[str: int] = interpret_moods()
```

Comment:
You don't have to be explicit here.

If `interpret_moods` has return type of `dict[str: int]`, then `moods` will implied as `moods: dict[str: int]`
```suggestion
    moods = interpret_moods()
```

Reply:
oh yeah, I suppose so.

Reply:
ahh I see.